jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -339,16 +328,11 @@ def mos_image_parser(app, data_obj, data_labels=None, share_image=0):
     if data_obj is None:
         return
 
-    if isinstance(data_obj, str):
```

Comment:
Tsk tsk, this overlaps with #783

Reply:
I'll try to review that PR tomorrow morning. The thought crossed my mind that you were changing the image parser anyway, but I figured I'd make it all consistent while I was at it, since your PR wasn't merged yet.

Reply:
The conflict is not fun though 😿